adapters/src/main/kotlin/com/krzykrucz/elesson/currentlesson/lessonprogress/adapters/rest/LessonProgressDto.kt

```diff
@@ -0,0 +1,23 @@
+package com.krzykrucz.elesson.currentlesson.lessonprogress.adapters.rest
+
+import com.krzykrucz.elesson.currentlesson.lessonprogress.usecase.LessonProgress
+import com.krzykrucz.elesson.currentlesson.shared.LessonTopic
+
+data class LessonProgressDto(
+    val semester: Int,
+    val className: String,
+    val subject: String,
+    val date: String,
+    val topic: LessonTopic?,
+    val status: String
+)
+
+fun LessonProgress.toDto() =
```

Comment:
not sure if it such extension function isn't too audacious when it comes to encapsulation, because of it, DTO leaks all over the place when `LessonProgresss` is used, maybe dependecy could be reversed -> `LessonProgressDto` has factory which takes `LessonProgresss` as parameter

Reply:
Agreed, done.